src/SFA.DAS.FAA.Database/Tables/SavedSearch.sql

```diff
@@ -4,7 +4,8 @@
     [DateCreated]                   datetime            NOT NULL,
     [LastRunDate]                   datetime            NULL,
     [EmailLastSendDate]             datetime            NULL,
-    [SearchParameters]  		    nvarchar(max)       NOT NULL    
+    [SearchParameters]  		    nvarchar(max)       NOT NULL,
+    [UnSubscribeToken]  		    nvarchar(max)       NOT NULL
```

Comment:
Just a thought, would the unsubscribe token field need an index on it, for lookup when someone comes to click on unsubscribe from the email?

Reply:
No need I guess, we will unpack the UnsubscribeToken using Data Protector to obtain the Unique identifier and fetch the record with that,

No need to pass the token to the APIM -> Inner Api to fetch the entity

Reply:
Ah ok so the token isn't random, gotcha.